appveyor.yml

```diff
@@ -6,15 +6,22 @@ branches:
 
 cache:
   # Note that we don't use cache for local Maven repository - see https://github.com/SonarSource/sonar-java/pull/525
-  - C:\ProgramData\chocolatey\bin -> appveyor.yml
-  - C:\ProgramData\chocolatey\lib -> appveyor.yml
-  - C:\bin\apache-maven-3.2.5 -> appveyor.yml
+  - C:\Users\appveyor\maven -> appveyor.yml
 
 install:
   - git submodule update --init --recursive
   - set MAVEN_VERSION=3.2.5
-  - choco install maven -version %MAVEN_VERSION%
-  - set PATH=%PATH%;C:\bin\apache-maven-%MAVEN_VERSION%\bin
+  # Note that indentation is important here:
+  - ps: |
+      Add-Type -AssemblyName System.IO.Compression.FileSystem
```

Comment:
@Godin we can achieve the same in an easier way, using `curl` and `unzip`, see https://github.com/dbolkensteyn/nemo-roslyn/blob/master/appveyor.yml#L14